tests/models/gemma/test_modeling_gemma.py

```diff
@@ -531,12 +531,12 @@ def test_flash_attn_2_equivalence(self):
                 assert torch.allclose(logits_fa, logits, atol=3e-3)
 
 
-@require_torch_gpu
 @slow
-@require_read_token
+@require_torch_gpu
```

Comment:
I think we are trying to move forward to use `require_torch_accelerator` instead of just `gpu` (despite we don't know if they will be run on other accelerators)

Reply:
I think it's fine to keep it torch_gpu here since we excplitly want to test it on GPUs